src/controllers/expensesController.js

```diff
@@ -0,0 +1,107 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+
+  if (!userId && !categories && (!from || !to)) {
+    res.send(expensesService.getAllExpenses());
+  }
```

Comment:
It's a potential bug. If the `userId` is not provided, `categories` is not provided, and either `from` or `to` is not provided, you return all expenses. However, if only `from` is provided and `to` is not (or vice versa), it is not clear what the expected behavior should be. Usually, in such cases, the date range should be considered as from the provided date to the current date (or from the start of the time to the provided date). You should clarify this point in your requirements and modify the code accordingly.